src/sentry/static/sentry/app/redux/reducers/workBatchDetails.js

```diff
@@ -12,19 +12,6 @@ const initialState = {
   workBatch: null,
 };
 
-function prepareWorkBatchForLocalWork(workBatch) {
```

Comment:
This I'll have to revisit in the next PR. I think the component that requires this should do this. But currently this breaks the  transition component (CLIMS-492). But since it was working with mocked data anyways (and I'm the one that's going to fix it) I felt that it was OK.